client/src/components/pages/Create.js

```diff
@@ -29,12 +29,44 @@ import "firebase/auth";
 import "./styles/Create.css";
 
 const mockTechStackData = {
-  languages: ["Java", "JavaScript", "C++", "C"],
-  frameworks: ["React", "Angular", "HTML", "CSS"],
-  workTools: ["Jira", "Asana", "Confluence", "Notion"],
+  languages: [
+    "Java",
+    "Python",
+    "C++",
+    "C",
+    "C#",
+    "JavaScript",
+    "TypeScript",
+    "HTML",
+    "CSS",
+    "R",
+    "MATLAB",
+    "Mathematica",
+    "SQL",
+  ],
+  frameworks: [
+    "React",
+    "Node",
+    "Angular",
+    "Bootstrap",
+    "MongoDB",
+    "Ruby on Rails",
+    "Google Cloud",
+    "AWS",
+    "Docker",
+    "Linux",
+    "Unix",
+  ],
+  workTools: ["GitHub", "Azure", "Jira", "Jupyter", "Notion", "Asana"],
   csConcepts: [
+    "Data Structures",
+    "Asynchronous programming",
+    "Agile Development",
```

Comment:
Like mentioned on call, we have a separate ticket to actually groom and populate this list. Some of these items don't belong on their respective categories (i.e. Linux and Unix). But this is a good start as well.

Reply:
For sure, I got these from our survey data (these were the most common languages used).